website/templates/project/project_header.mako

```diff
@@ -122,6 +122,10 @@
                     <br />Forked from <a class="node-forked-from" href="/${node['forked_from_id']}/">${node['forked_from_display_absolute_url']}</a> on
                     <span data-bind="text: dateForked.local, tooltip: {title: dateForked.utc}"></span>
                 % endif
+                % if node['is_registration']:
```

Comment:
On second look, I think this is ok because it keeps consistency with the rest of this file.